keps/sig-architecture/2140-receipts/README.md

```diff
@@ -0,0 +1,266 @@
+
+The receipts would need to be created in the [k/enhancements][k-enhancements] repository. Once a KEP Author/Assignee creates a receipt on behalf of a SIG, it is understood that the feature has been soft opted-in to the release. The opt-in would be finalised only after the Enhancements Team verifies the laid out acceptance criteria.
+
+The Enhancements Team will continuously monitor the filed receipts during the Enhancements Collection period to verify the graduation requirements. The Enhancements Team will have the power to remove enhancements from the release if the team feels they are not conforming requirements. The team can ask the KEP Assignee to file an exception if Enhancements Freeze deadline has been passed.
+
+After collection of Enhancements is completed, KEP Assignees will need to update their receipts with the implementation PRs and documentation PRs (if any). KEP Assignees can update the receipt with the relevant PRs before Enhancements Freeze as well. The Enhancements Team will monitor the mentioned PRs for Code Freeze completion requirements. The team will have the right to ask the assignee to file for an exception or counsel the assignee whether they want to bump graduation of an enhancement to the next release cycle.
+
+The RT Docs team will monitor the mentioned documentation PRs to see if it conforms to the Docs deadline requirements of that release cycle.
+
+[See below][#design-details] for the receipt format and the process.
+
+### User Stories
+
+TODO: Mention the personas who are benefited through the improvements listed in this KEP
+TODO: Should we rewrite these into user story format? i.e. "As a SIG tech lead, I want to graduate a KEP in a specific release"
+
+#### KEP Authors
+
+##### Proposing enhancements
+
+* Specify New KEP for Release
+* Specify Graduation phase for existing KEP
+* Link / Report k/k PRs for a given Enhancment
+* Indicate k/w PRs for a given Enhancement
+
+##### Filing an exception to Enhancements/Code Freeze
+
+* Request Exception for Enhancement Freeze
+* Request Exception for Code Freeze
+
+#### API Reviewers
+
+* Review / Approve Exception Requests that need API Review
+    * Ensure reviewer bandwidth considered
+    * Priortize areas for upcoming releases
+
+#### SIG Chair / TLs / PMS
+
+* Review Open/Proposed KEPs for my SIG
+    * Help plan reviews / dependencies
+    * Help identify major themes / release blogs
+* Review At Risk KEPS for my SIG
+    * Help Prioritize last minute reviews
+* Review Exceptions for my SIG
+    * Determine release blocking / major importance
+* Review Slipped Enhancements to support planning
+    * Determine what phase it missed
+    * Identify multiple slips / misses
+
+#### Enhancements Subteam
+
+* Pre enhancements Freeze (Enhancements Collection)
+  * Collect intents of KEP authors to graduate a KEP
```

Comment:
I am planning to define what is collecting intent in the implementation section as well so that things are crystal clear. And, the enhancements team and/or other subteams of the release team do not need to do most of the leg work as is done today.